### DISCUSSION TITLE
Install Problem of Python interface (experimental)

### DISCUSSION THREAD

**Discussion by soumyodiptak** [galacticusorg/galacticus#554] [Q&A]
Hi, I am a graduate student at University of New Mexico working with Prof. Cyr-Racine. I would like to use Galacticus for my research. I have MacBook Pro with M2 Max chip and macOS: 14.2.1. 

I followed the instructions for installation from "Using Precompiled Binaries on MacOS" page, and the installation was done successfully.
Now I am following the page "Python interface (experimental)". When I entering "make -j16 GALACTICUS_BUILD_OPTION=lib libgalacticus.so", I am getting errors. I have attached the snapshot of the error. Please check it and let me know what I should do.

Best Regards,
Soumyodipta Karmakar

<img width="1728" alt="Screenshot 2024-02-12 at 16 27 30" src="https://github.com/galacticusorg/galacticus/assets/40052609/ff96fc52-c8b7-4779-b3d7-841786d0a79e">

**Comment by abensonca** [comment]
Hi Soumyodipta. Building from source on MacOS is still somewhat difficult. Did you try following the instructions [here](https://github.com/galacticusorg/galacticus/wiki/Installation-from-source-on-MacOS)? We had some success installing from source on an M2 chip a few months ago, but the detailed instructions have not yet been written up unfortunately.

Do you absolutely need the Python interface? It is very experimental and quite limited in functionality at this point - so I would not suggest using it unless you absolutely have to.
